.github/workflows/e2e-app-store.yml

```diff
@@ -3,6 +3,41 @@ on:
   workflow_call:
 env:
   NODE_OPTIONS: --max-old-space-size=4096
+  ALLOWED_HOSTNAMES: ${{ vars.CI_ALLOWED_HOSTNAMES }}
```

Comment:
Putting these vars at the top of the file helps with making sure child jobs always have them set